Documentation/contributor-guide/features.md

```diff
@@ -45,6 +46,38 @@ Any new enhancements to the etcd are typically added as an experimental feature.
     - Add a CHANGELOG entry.
 - At least two maintainers must approve feature requirements and related code changes.
 
-### Graduating an Experimental feature to Stable (TODO - spzala)
+### Graduating an Experimental feature to Stable
 
-### Deprecating a feature (TODO - spzala)
+It is important that experimental features don't get stuck in that stage. They should be revisited and moved to the stable stage following the graduation steps as described here.
+
+#### Locate graduation candidate
+Decide if an experimental feature is ready for graduation to the stable stage.
+- Find an issue that was used to enable the experimental feature initially. One way to find such issues is to search for issues with `type/feature` and `experimental` labels.
+- Fix any known open issues against the feature.
+- Make sure the feature was enabled for at least one previous release. Check the PR(s) reference from the issue to see when the feature related code changes were merged.
+
+#### Provide implementation
+If an experimental feature is found ready for graduation to the stable stage, open a Pull Request (PR) with the following changes.
+- Add robust e2e tests if not already provided.
+- Add a new stable feature name identical to the experimental feature name but without the `--experimental` prefix.
+- Deprecate the experimental feature following the [feature deprecation policy](#Deprecating-a-feature).
+- Implementation must ensure that both the graduated and deprecated experimental features work as expected. Note that both these features will co-exist for the timeframe described in the feature deprecation policy.
```

Comment:
Just both the experimental and stable flags co-exist, and actually they are actually the same feature. If users are still using the experimental flag, we should print a warning log.

Reply:
@ahrtr yes, and that warning would be "Deprecated. Use <the stable feature> instead."

Reply:
ack